packages/marshal/docs/marshal-vs-store-level.md

```diff
@@ -0,0 +1,95 @@
+
+On the right is the *JS* level, explaining how these map onto JavaScript language. This mapping determines how JavaScript values round trip or not through the protocol. Only hardened JavaScript values can be passable. The mapping of protocol concepts to JavaScript should serve as an example of how to map the protocol onto the concepts of other languages.
+
+|                | Store Level        | Marshal level                       | JS                                                    |
+| -------------- | ------------------ | ----------------------------------- | ----------------------------------------------------- |
+| Classification | `keyStyleOf(p)`<br>`patternStyleOf(p)` | `passStyleOf(p)`<br>`getInterfaceOf(j)` | `typeof j`                    |
+| Type Testing   | `isKey(p)`         | `isOnlyData(p)`<br>`isStructure(p)` | `isPrimitive(j)`<br>`Array.isArray(j)`                |
+| Equivalence    | `sameKey(k,k)`     | `sameStructure(s,s)`                | `j === j`<br>`Object.is(j,j)`<br>`sameValueZero(j,j)` |
+| Ordering       | `compareKeys(k,k)` | `comparePassables(p,p)`             | `j <= j`<br>`[...js].sort(fullCompare(j,j))`          |
+
+Where the parameter
+   * `j` is for any JavaScript value.
+   * `p` is for any `Passable`, a subset of JavaScript values.
+   * `s` is for any `Structure`, a subset of passables.
+   * `k` is for any `Key`, a subset of structures.
+
+
+|            | `keyStyleOf`            | `passStyleOf`          | `typeof`                                                     |
+| ---------- | ----------------------- | ---------------------- | ------------------------------------------------------------ |
+| Primitives | js primitives + `null`  | js primitives + `null` | `undefined` `boolean` `bigint`<br>`number` `string` `symbol` |
+| Containers | `copyArray` `copyRecord`<br>`copySet` `copyMap` | `copyArray` `copyRecord`<br>`copyTagged` | `object`           |
+| Behavior   | `remotable`             | `remotable`            | `function`                                                   |
+| Others     | No keys<br>See patterns | `error` `promise`<br>`metaTagged` |                                                   |
+
+
+
+## The Marshal level organized by `passStyleOf`
+
+We organize according to the classification from<br>
+**`passStyleOf(p: Passable) => PassStyle`**<br>
+where the returned `PassStyle` is one of a fixed enumeration of strings, similar to the strings returned by JavaScript's `typeof`.
+
+`PassStyle`:
+  * **Primitive styles**: `"undefined"`, `"null"`, `"boolean"`, `"bigint"`, `"number"`, `"string"`, `"symbol"`
+  * **Data styles**: The primitive styles + `"copyArray"`, `"copyRecord"`, `"copyTagged"`
+  * **Structure styles**: The data styles + `"remotable"`
+  * **`PassStyle`**: The structure styles + `"error"`, `"promise"`, `"metaTagged"`
```

Comment:
It's a little confusing to have `PassStyle` naming both the overall type and one of its subcategories. Maybe something like:

`PassStyle`:
* Structure style:
    * Data style:
        * Primitive style: `"undefined"`, `"null"`, ...
        * `"copyArray"`, `"copyRecord"`, `"copyTagged"`
    * `"remotable"`
* `"error"`, `"promise"`, or `"metaTagged"`

Reply:
Good idea! Done. Resolving.